_pages/how-we-work/tools/trello.md

```diff
@@ -40,23 +40,9 @@ People use Trello to track ideas from conception through execution. It's common
-
-- [Show card numbers on cards](https://chrome.google.com/webstore/detail/trello-card-numbers/kadpkdielickimifpinkknemjdipghaf) makes it easy for people to refer to cards definitively in remote screen-sharing meetings. (Just make sure everyone realizes these are card IDs, not the card's priority!)
-
-- [Show labels as text as well as color](https://chrome.google.com/webstore/detail/card-color-titles-for-tre/hpmobkglehhleflhaefmfajhbdnjmgim) in order to make it really clear to your team what your label colors represent. (Once everyone knows them cold, you can turn this off to avoid visual noise.)
-
-- Need to export your entire board for use in another tool? Normally you'd have to buy a paid Trello plan for this, but if you don't need the other features in the paid plans, install the [Export for Trello](https://chrome.google.com/webstore/detail/export-for-trello/nhdelomnagopgaealggpgojkhcafhnin) Chrome extension to get your board data into CSV and JSON easily.
-
-- Tracking stories for multiple sub-components/teams/projects on the same board? Install the [Projects for Trello](https://chrome.google.com/webstore/detail/projects-for-trello/mholjhodapabhdbchonjjoecmfhobfoa) extension, then prefix card names with {component} to put a badge on the card referring to a sub-component or project.
-
-- Doing Kanban? You'll want the [CardCounter for Trello](https://chrome.google.com/webstore/detail/cardcounter-for-trello/miejdnaildjcmahbhmfngfdoficmkdhi) Chrome extension. Document your Work-in-Progress (WIP) limit for a column in a bracket at end of the title, for example, "In Progress [3]". Trello will show how many cards you have in progress and the marker will turn red when you exceed the limit. Looking for a Continuous Flow Diagram or other metrics? Check out [Trello-CFD](http://trello-cfd.azurewebsites.net/) for the bare minimum and [Corello](https://getcorrello.com/) for a more deluxe option.
-
-- Using estimates as part of your process? Install the [Points for Trello](https://chrome.google.com/webstore/detail/points-for-trello/mkcpchladphoadhaclmnlphhijboljjk?hl=en) Chrome extension. Then when you edit the title for cards, you can append/prepend the estimate in parentheses, for example, "(3) Do this thing". If you use numbers, then the total will be shown at the top of the column. Also want to track points burndown? Use the [Burndown for Trello](https://www.burndownfortrello.com/). (If you also want to track effort spent, use the [Scrum for Trello](https://chrome.google.com/webstore/detail/scrum-for-trello/jdbcdblgjdpmfninkoogcfpnkjmndgje) Chrome extension in place of both of these.)
+At this time, no Chrome extensions for Trello have been approved for use. You can [complete this form to request an extension for review](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSeviBt7bnriQz3iQRn1dUyg-8KUR7jRgG5eJbc7nHxa1D_ciw/viewform?pli=1).
```

Comment:
If this is the case, then why not just collapse up all of this? Something like:
"Chrome extensions can dramatically increase the usability and functionality of Trello. However, at this time, no Chrome extensions for Trello have been approved for use. If you would like to use implement an extension for Trello on your team, you'll need to [complete this form to request an extension for review](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSeviBt7bnriQz3iQRn1dUyg-8KUR7jRgG5eJbc7nHxa1D_ciw/viewform?pli=1)."

Or something like that. I'm just not sure it makes sense to say "Trello extensions are awesome. Be sure they're approved. They're not." which is kind of how it reads now.